tests/mtd_mapper/Makefile

```diff
@@ -1,6 +1,7 @@
 include ../Makefile.tests_common
 
 USEMODULE += mtd_mapper
+USEMODULE += mtd_write_page
```

Comment:
Kconfig needs an update too - please squash directly

Reply:
Sure ... always forgetting this.

I hope we'll get rid of this redundancy soon ;-)

Reply:
Sure ... always forgetting this.

I hope we'll get rid of this redundancy soon ;-)